websocket_test.go

```diff
@@ -564,11 +564,11 @@ func TestWebSocketTLS_Error(t *testing.T) {
 	require.NotEqual(t, "", log.GetStdErr())
 }
 
-// TestWebSocket_Streaming performs 3 test cases.
+// TestWebSocket_Streaming_normal performs 3 test cases.
```

Comment:
I wonder if this would be a good time to separate these out into 3 test cases? I'm also not clear why the rest for your new code is in processor_test.go and not here. (Not saying you should change, just want to understand your reasoning.)

Reply:
Most of the new work is the implementation of `ProcessClientStreamRequest()` in processor.go, where the changes in websocket.go are mainly refactoring. So I primarily wanted to unit test this new function. 
But yes, let's also update websocket_test.go. Increasing test coverage won't hurt.